.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,7 @@
 
 ## What did you implement:
 
-***Implementing Issue:*** #12345
+closes #12345
```

Comment:
Could we uppercase this to `Closes` as everything else which is in a newline is uppercased as well?